wgpu/gui/glfw.py

```diff
@@ -62,13 +34,17 @@ class GlfwWgpuCanvas(BaseCanvas):
     """ A canvas object wrapping a glfw window.
     """
 
+    # See https://www.glfw.org/docs/latest/group__window.html
+
     def __init__(self, *, size=None, title=None):
         super().__init__()
+
+        # Handle inputs
         if size:
-            width, height = size
+            size = float(size[0]), float(size[1])
         else:
-            width, height = 256, 256
-        title = title or ""
+            size = 640, 480
```

Comment:
these aren't floats, so why do you explicitly cast to floats in line 44?

Reply:
yeah, there was a bit of historic stuff there. The cast to float functioned as a typecheck, but this is (in the current code) already done a bit below. I'm cleaning this up.